README.md

```diff
@@ -1,5 +1,14 @@
-## Weather Project
+# Redux Weather App
 
-This project has been created by a student at Parsity, an online software engineering course. The work in this repository is wholly of the student based on a sample starter project that can be accessed by looking at the repository that this project forks.
+Search a city name and get back three graphs displaying the temperature, pressure, humidity, and corresponding averages for that city over the next 5 days. This app was built with React using Vite, Redux, and utilizes the [OpenWeather API](https://openweathermap.org/).
 
-If you have any questions about this project or the program in general, visit [parsity.io](https://parsity.io/) or email hello@parsity.io.
+## Run Locally
+
+To run locally, fork and clone this repository. `cd` into the directory you created and execute the following commands:
+
+```
+npm install
```

Comment:
nice job on the readme! it is great to put all instructions on how to run the app in here.